scene/gui/tree.cpp

```diff
@@ -247,31 +247,25 @@ void TreeItem::_propagate_check_through_parents(int p_column, bool p_emit_signal
 		return;
 	}
 
-	bool all_unchecked_and_not_indeterminate = true;
-	bool any_unchecked_or_indeterminate = false;
+	bool has_checked = false;
```

Comment:
I'd keep the naming `any_*`

Reply:
May I name them like this:
```
bool any_checked = false;
bool any_unchecked = false;
bool any_indeterminate = false;
```

Reply:
It's either way, was just suggesting, but I prefer `any` here